MeshLib/Mesh.cpp

```diff
@@ -413,6 +413,13 @@ PropertyVector<int> const* materialIDs(Mesh const& mesh)
 std::unique_ptr<MeshLib::Mesh> createMeshFromElementSelection(
     std::string mesh_name, std::vector<MeshLib::Element*> const& elements)
 {
+    if (elements.empty())
+    {
+        ERR("Empty element selection. Function createMeshFromElementsSelection "
+            "can not create mesh '%s'.",
+            mesh_name.c_str());
+        return nullptr;
```

Comment:
The `return nullptr` causes several benchmarks to fail. In the current master sometimes meshes with zero nodes and zero elements are created automatically from geometries. I doubt these meshes are used later on for something.
@endJunction, @wenqing: Are there cases where such meshes are useful during a simulation? **For discussion**: Is a mesh with zero nodes and zero elements a valid mesh?

Reply:
Zero nodes/elements mesh is a perfectly fine mesh, in my opinion, just a trivial one. Happens for partitioned meshes, for example, if boundary geometry doesn't have an intersection with that partition, but those result in .bin files. There should be no trivial meshes for serial cases used as input.
I've to admit, that I didn't look into the failing benchmarks (yet).